legal-api/src/legal_api/resources/v2/colin.py

```diff
@@ -0,0 +1,42 @@
+# Copyright © 2021 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Searching on a namerequest.
+
+Provides a proxy endpoint to retrieve corp's information.
+"""
+from flask import Blueprint, abort, current_app, jsonify, make_response
+from flask_cors import cross_origin
+
+from legal_api.services import colin
+
+
+bp = Blueprint('COLIN2', __name__, url_prefix='/api/v2/colin')
+
+
+@bp.route('/<string:corp_num>', methods=['GET'])
+@cross_origin(origin='*')
+def get(corp_num):
```

Comment:
This endpoint needs to be secure.  

Check that the api consumer for this endpoint has the `COLIN_SVC_ROLE`.

Reply:
This endpoint will be call by NameX